readme.md

```diff
@@ -375,6 +375,10 @@ The processing of the optional field `quote` is currently unspecified.
 If the attester requires payment to issue this credential, the `quote` MUST be present.
 If the attester does not require payment to issue this credential, the `quote` MUST NOT be present.
 
+If the `owner` DID URI is provided and the extension controls this DID, then it SHOULD NOT be changed. 
+Legacy attesters provide meaningless values in this field, so if the extension does not control this DID
+then the value MUST be changed to the DID the user chooses.
+
```

Comment:
I'm a bit confused by the meaning of 'changed' in this paragraph. In the `submit-terms` message the attester either provides the owner value or does not provide it. When is it supposed to change the value?

Reply:
I understand your confusion but not sure how to express this more clearly. Maybe like this?

> If the `owner` DID URI is provided and the extension controls this DID, then the value of `owner` in the response SHOULD be the same.
> Legacy attesters provide meaningless values in this field, so if the extension does not control this DID, then the value of `owner` in the response MUST be a DID URI that the extension controls.